### ISSUE TITLE
 how can i use?

### ISSUE THREAD

**Issue by engelsizmuzik** [Happy0/react-native-bluetooth-socket-bridge#1]
Hello,

Are you going to publish a guide for this library?

**Comment by Happy0** [comment]
Yeah, I will, once I have time. I recently started a job so my main focus is on that for now.

In the meantime, I suspect that https://github.com/rusel1989/react-native-bluetooth-serial would meet most people's needs.

This library was created for the scuttlebutt project (https://www.scuttlebutt.nz) to to allow a nodejs thread to use Bluetooth functionality in android via Unix sockets, which is probably a rare use case.

It's used by https://github.com/happy0/ssb-mobile-bluetooth-manager

Which is being used for the (work in progress) functionality for manyverse ( https://gitlab.com/staltz/manyverse/merge_requests/55 )

**Comment by anilslabs** [comment]
@Happy0 Can you please provide a working example snippet for this.
